src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,26 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.models.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao =
+                (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+        Manufacturer bars = manufacturerDao
+                .create(new Manufacturer("Bars", "Ukraine"));
+        Manufacturer subaru = manufacturerDao
+                .create(new Manufacturer("Subaru", "Japan"));
+        System.out.println(manufacturerDao.get(1L));
```

Comment:
```suggestion
        System.out.println(manufacturerDao.get(bars.getId()));
```
dont use hardcoded id values, get it from create method, this way your code works even after several runs
same with delete